specification/sql/resource-manager/Microsoft.Sql/preview/2023-05-01-preview/JobAgents.json

```diff
@@ -354,6 +410,14 @@
       "description": "An update to an Azure SQL job agent.",
       "type": "object",
       "properties": {
+        "identity": {
+          "$ref": "#/definitions/JobAgentIdentity",
+          "description": "Managed identity assigned to job agent"
+        },
+        "sku": {
+          "$ref": "../../../common/v1/types.json#/definitions/Sku",
```

Comment:
>/v1/types.json#/definitions/Sku [](http://example.com/codeflow?start=34&length=31)

its also there in v5, so I'd just suggest using the general purpose v5 definition or... (can't believe I'm saying this but) declaring your own sku type. Because often RPs really can document their own SKU type much better than a generic documentation would.

[https://github.com/Azure/azure-rest-api-specs/blob/main/specification/common-types/resource-management/v5/types.json](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/common-types/resource-management/v5/types.json)

Reply:
This seems to have been autopopulated, and seems to be using on other places of the repo
https://github.com/Azure/azure-rest-api-specs/blob/release-sql-Microsoft.Sql-2023-05-01-preview/specification/sql/resource-manager/Microsoft.Sql/preview/2023-05-01-preview/Databases.json